TensorKitSectors/src/precompile.jl

```diff
@@ -0,0 +1,32 @@
+"""
+    precompile_sector(I::Type{<:Sector})
+
+Precompile common methods for the given sector type.
+"""
+function precompile_sector(::Type{I}) where {I<:Sector}
+    precompile(Nsymbol, (I, I, I))
+    precompile(Fsymbol, (I, I, I, I, I, I))
+    precompile(Rsymbol, (I, I, I))
+    precompile(Asymbol, (I, I, I))
+    precompile(Bsymbol, (I, I, I))
+
+    precompile(⊗, (I,))
+    precompile(⊗, (I, I))
+    precompile(⊗, (I, I, I))
+    precompile(⊗, (I, I, I, I))
+
+    precompile(FusionStyle, (I,))
+    precompile(BraidingStyle, (I,))
+
+    precompile(dim, (I,))
+    precompile(sqrtdim, (I,))
+    precompile(isqrtdim, (I,))
```

Comment:
I don't remember how I came to call this `isqrtdim` with `i` for `inv`(erse). It seems like `Base.isqrt` is for integer square root. I wouldn't be opposed to making this `invsqrtdim` moving forward.